rmgpy/data/kinetics/depository.py

```diff
@@ -173,7 +173,7 @@ def load(self, path, local_context=None, global_context=None):
             for product in products.split('+'):
                 product = product.strip()
                 if product not in speciesDict:
-                    raise DatabaseError('Species {0} in kinetics depository {1} is missing from its dictionary.'.format(product, self.label))
+                    raise DatabaseError('Species {0} in kinetics depository {1} in reaction {2} is missing from its dictionary.'.format(product, self.label,rxn_string))
```

Comment:
Would it be more helpful to provide the entry index in addition or instead of the reaction string?